### PR TITLE
Update wheels build + Update linux build tag for wheels

### PR DESCRIPTION
**Pull request by Vyacheslav-Smirnov**
- Update wheels build procedure - align with other packages; current build via conda output does not support required additional options
 - Use manylinux1 for Python 3.6
 - Use manylinux2014 for Python > 3.6

### CODE REVIEW

**Comment by Vyacheslav-Smirnov** [comment]
Mac build fails on gold/2021 branch also. I do not think that it is caused by current changes.
So I propose to merge this PR.

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -13,15 +13,12 @@ build:
...
     - SDC_CONFIG_PIPELINE_SDC
+    - WHEELS_OUTPUT_FOLDER
 
 requirements:
   build:
     - {{ compiler('c') }}      # [not osx]
     - {{ compiler('cxx') }}    # [not osx]
-    - wheel
-    - python
-    - numba   {{ NUMBA_VERSION }}
```
Comment (kozlov-alexey) [inline conda-recipe/meta.yaml:23]: Numba recently became a build dependency for sdc, so it's better to keep it.

File: conda-recipe/meta.yaml [hunk 2]
```diff
@@ -45,24 +42,6 @@ test:
   imports:
     - sdc
 
-outputs:
```
Comment (kozlov-alexey) [inline conda-recipe/meta.yaml:48]: Why to remove 'outputs' section at all?

File: conda-recipe/meta.yaml (hunk 1 above)
Comment (Vyacheslav-Smirnov) [reply conda-recipe/meta.yaml:23]: Ok, let's keep numba here

File: conda-recipe/meta.yaml (hunk 2 above)
Comment (Vyacheslav-Smirnov) [reply conda-recipe/meta.yaml:48]: One of the purpose of this PR is to align wheels build procedure with others packages.
Here was used conda output - it does not suits for us due to we cannot set additional arguments for wheels build.
That is why output section is removed and wheel build lines are added in build.sh and bld.bat

**Pull request merged**